### PR TITLE
0.9.0 release notes

### PR DESCRIPTION
**Pull request by qiaojialin**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 -->
 
-# Apache IoTDB (incubating) 0.8.0
+# Apache IoTDB (incubating) 0.9.0
 
 This is the first official release of Apache IoTDB after joining the Incubator.
```
Comment (LeiRui) [inline RELEASE_NOTES.md]: I would expect the notes to contain all history notes, not just the latest ones. In other words, I would prefer to see release notes of 0.8.0, 0.8.1 and 0.9.0 appear on the same page.
With that said, "This is the first official release of Apache IoTDB after joining the Incubator." is talking about 0.8.0 :)

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -33,138 +33,72 @@ This is the first official release of Apache IoTDB after joining the Incubator.
...
+* IOTDB-241	Add query and non query interface in session
+* IOTDB-223	Add a TsFile sketch tool
+* IOTDB-158	add metrics web service
+* IOTDB-187	Enable to choose storage in local file system or HDFS
+* IOTDB-199	Add a log visulization tool 
+* IoTDB-174	Add interfaces for querying device or timeseries number
+* IOTDB-173	add batch write interface in session
+* IOTDB-151	support number format in timeseries path
+* IOTDB-294	online upgrade from 0.8.0 to 0.9.0
+* modify print function in AbstractClient
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply RELEASE_NOTES.md]: remove that in the new feature

File: RELEASE_NOTES.md (hunk 1 above)
Comment (qiaojialin) [reply RELEASE_NOTES.md]: fixed

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -19,17 +19,115 @@
...
-* Bug Fixes
+* IOTDB-266 NullPoint exception when reading not existed devices using ReadOnlyTsFile
+* IOTDB-264	Restart failure due to WAL replay error
+* IOTDB-240	Fix unknown time series in where clause
+* IOTDB-244	Fix bug when querying with duplicated columns
+* IOTDB-168&169 Fix a bug in export-csv tool and fix compatibility of timestamp formats in exportCsv, client display and sql
+* IOTDB-174	Fix querying timeseries interface cannot make a query by the specified path prefix
+* IOTDB-195	Using String.getBytes(utf-9).length to replace string.length() in ChunkGroupMetadata for supporting Chinese
+* IOTDB-211	Use "%IOTDB_HOME%\lib\*" to refers to all .jar files in the directory in start-server.bat
+* IOTDB-244 Fix bug when querying with duplicated columns
```
Comment (LeiRui) [inline RELEASE_NOTES.md]: IOTDB-244 appears twice.

Comment (qiaojialin) [reply RELEASE_NOTES.md]: fixed

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -19,17 +19,115 @@
...
+# Apache IoTDB (incubating) 0.8.1
+
+This is a bug-fix version of 0.8.0
+
+- [IOTDB-172] bug in updating startTime and endTime in TsFileResource
+- Abnormal publishing of sequence and unsequence data folders in DirectoryManager
+- fix a bug in TimeRange's intersects function
+- [IOTDB-202] fix tsfile example data type
+- [IOTDB-195] Bug about 'serializedSize' in ChunkGroupMetaData.java (for Chinese string)
+- [IOTDB-242] fix mvn integration-test failed because the files in the target folder changes
```
Comment (LeiRui) [inline RELEASE_NOTES.md]: Forgive me for nitpicking, but please keep the format consistent as you can.
I see that some are surrounded with [] such as [IOTDB-202] and some are not. 
Some are indented with short spaces and some are with longer spaces. For example:
image

Comment (LeiRui) [reply RELEASE_NOTES.md]: Also, sorting the item by their IOTDB-XXX number is a good habit.

Comment (qiaojialin) [reply RELEASE_NOTES.md]: Thanks, I do not notice it when using the idea

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -19,41 +19,140 @@
...
+# Apache IoTDB (incubating) 0.9.0
 
-This is the first official release of Apache IoTDB after joining the Incubator.
+## New Features
+
+* IOTDB-143 Compaction of data file
+* IOTDB-208 Add Bloom filter in TsFile
+* IOTDB-205 Support storage-group-level Time To Live (TTL)
+* IOTDB-198 Add sync module (Sync TsFiles between IoTDB instances)
+* IoTDB 226 Hive-TsFile connector
```
Comment (LeiRui) [inline RELEASE_NOTES.md]: IoTDB-226

Comment (qiaojialin) [reply RELEASE_NOTES.md]: fixed

**Comment by LeiRui** [comment]
Better rebase before merge.

**Pull request merged**